### ISSUE TITLE
Doc: Release Notes, Change Logs, v0.2, v0.3 and master branch docs

### ISSUE THREAD

**Issue by paulocheque** [futurepress/epub.js#471]
Is there any doc about all of this?

I found some basic info in the README and some title in the release package: https://github.com/futurepress/epub.js/releases/tag/alpha-0.3.1

Is there a NPM package for the v0.3?

Thanks in advance

**Comment by Asido** [comment]
From what I see branch v0.3 is where the development is happening, although it still needs more cooking to be ready for production imo.

**Comment by fchasen** [comment]
Yes, still needs a little more time before it is ready to switch to but getting very close.

I'll compile a roadmap next week on github, sorry about the lack of communication on that. I'm planning to use "projects" to track the remaining task before it is ready for release.
